ui/tests/integration/components/clients-history-test.js

```diff
@@ -0,0 +1,67 @@
+import { module, test } from 'qunit';
+import EmberObject from '@ember/object';
+import { render } from '@ember/test-helpers';
+import { setupRenderingTest } from 'ember-qunit';
+import hbs from 'htmlbars-inline-precompile';
+
+module('Integration | Component | client count history', function(hooks) {
+  setupRenderingTest(hooks);
+  hooks.beforeEach(function() {
+    let model = EmberObject.create({
+      config: {},
+      activity: {},
+    });
+    this.model = model;
+    this.tab = 'history';
+  });
+
+  test('it shows empty state when disabled and no data available', async function(assert) {
+    Object.assign(this.model.config, { enabled: 'Off', queriesAvailable: false });
+    await render(hbs`<Clients::History @tab={{tab}} @model={{model}} />`);
+
+    assert.dom('[data-test-component="empty-state"]').exists('Empty state exists');
+    assert.dom('[data-test-empty-state-title]').hasText('Data tracking is disabled');
+  });
+
+  test('it shows empty state when enabled and no data available', async function(assert) {
+    Object.assign(this.model.config, { enabled: 'On', queriesAvailable: false });
+    await render(hbs`<Clients::History @tab={{tab}} @model={{model}} />`);
+
+    assert.dom('[data-test-component="empty-state"]').exists('Empty state exists');
+    assert.dom('[data-test-empty-state-title]').hasText('No monthly history');
+  });
+
+  test('it shows empty state when data available but not returned', async function(assert) {
+    Object.assign(this.model.config, { queriesAvailable: true });
+    await render(hbs`<Clients::History @tab={{tab}} @model={{model}} />`);
+    assert.dom('[data-test-pricing-metrics-form]').exists('Date range form component exists');
+    assert.dom('[data-test-pricing-result-dates]').doesNotExist('Date range form result dates are not shown');
+    assert.dom('[data-test-component="empty-state"]').exists('Empty state exists');
+    assert.dom('[data-test-empty-state-title]').hasText('No data received');
+  });
+
+  test('it shows warning when disabled and data available', async function(assert) {
+    Object.assign(this.model.config, { queriesAvailable: true, enabled: 'Off' });
+    await render(hbs`<Clients::History @tab={{tab}} @model={{model}} />`);
+
+    assert.dom('[data-test-pricing-metrics-form]').exists('Date range form component exists');
+    assert.dom('[data-test-tracking-disabled]').exists('Flash message exists');
+    assert.dom('[data-test-tracking-disabled] .message-title').hasText('Tracking is disabled');
+  });
+
+  test('it shows data when available from query', async function(assert) {
```

Comment:
These two test files look exactly the same, are there any differences or should they be testing the exact same thing?

Reply:
Some tests are different, such as making sure date component is not shown in current tab and also the input data is slightly different (compare last test in both files).